examples/prefect/model_data_benchmark.py

```diff
@@ -0,0 +1,23 @@
+import pickle
+import sys
+import time
+
+import numpy as np
+from sklearn import base
+
+import nnbench
+
+
+@nnbench.benchmark(name="Model size (bytes)", tags=("model-meta",))
+def modelsize(model: base.BaseEstimator) -> int:
+    model_bytes = pickle.dumps(model)
+    return sys.getsizeof(model_bytes)
+
+
+@nnbench.benchmark(name="Inference time (s)", tags=("model-meta",))
+def inference_time(model: base.BaseEstimator, X: np.ndarray, n_iter: int = 100) -> float:
+    start = time.perf_counter()
+    for i in range(n_iter):
+        _ = model.predict(X)
+    end = time.perf_counter()
+    return (end - start) / n_iter
```

Comment:
We time each benchmark anyway, so we technically have this in the bag. Any idea how could we give the option to use the execution time as the target metric instead?

(Otherwise, it's fine - though the `time_ns` metric then becomes superfluous.)

Reply:
Good point. I'll think of another metadata metric we can use.

Reply:
Though the `time_ns ` times the whole benchmark. Here we know the time it takes for the plain inference. Both are different (esp. with more complex benchmarks) but of interest, no?

Reply:
Yeah, what I meant is that both should be the same if the benchmark consists only of "_ = model.predict(X)".

But that's statistically unstable, so please leave it as is.